js/packages/mobile-wallet-adapter-protocol/src/types.ts

```diff
@@ -78,7 +78,7 @@ export interface GetCapabilitiesAPI {
     >;
 }
 export interface ReauthorizeAPI {
-    reauthorize(params: { auth_token: AuthToken }): Promise<AuthorizationResult>;
+    reauthorize(params: { auth_token: AuthToken; identity: AppIdentity }): Promise<AuthorizationResult>;
```

Comment:
can we make `identity` optional here?

Reply:
We chatted about this. I'm opposed (when dApps don't identify themselves, it leads to a bad UX), and in any case the `identity` _object_ is not optional, so to make it optional here would require a change to the spec.